e2e-tests-rust/tests/lib.rs

```diff
@@ -369,15 +377,28 @@ async fn set_chain_id() -> anyhow::Result<()> {
 async fn cli_no_cors() -> anyhow::Result<()> {
     let mut headers = HeaderMap::new();
     headers.insert(ORIGIN, HeaderValue::from_static("http://some.origin"));
-    
-    // Verify all origins are allowed by default
+    headers.insert(HOST, HeaderValue::from_static("http://some.origin"));
+
+    // Verify cors is disabled by default
```

Comment:
CORS should not be disabled by default.

Reply:
Yes sorry for the confusion, let's continue this in the main thread